src/scripts/main.js

```diff
@@ -1 +1,104 @@
 'use strict';
+
+function promise1() {
```

Comment:
let's make all promises variables and write all code inside Promise constructor
```suggestion
const firstPromise = new Promise((resolve, reject) => {
```